acme/tests/crypto_util_test.py

```diff
@@ -187,11 +228,19 @@ def setUp(self):
 
```

Comment:
Can we rename this class to something like `GenSSCertTest` and update the docstring with the expanded scope here?